Dockerfile

```diff
@@ -24,10 +26,11 @@ RUN apt update && \
   apt install -y --no-install-recommends git && \
   git clone https://github.com/paritytech/substrate-telemetry.git substrate-telemetry && \
   cd substrate-telemetry && \
-  git checkout 0a89382127b9fb1b95d144cae816c46582975e93 && \
-  yarn && yarn build:all
+  git checkout  ${SUBSTRATE_TELEMETRY_TAG} && \
```

Comment:
they were also different tags !!! ( unless it was done by purpose )

Reply:
Not afaik :) we were more concerned about backend changes and forgot about the frontend...